src/ts/builders/open-url-with-default-browser-command-builder.ts

```diff
@@ -1,6 +1,7 @@
 export class OpenUrlWithDefaultBrowserCommandBuilder {
     public static buildWindowsCommand(url: string): string {
-        return `start explorer "${url}"`;
+        // return `start explorer "${url}"`;
+        return `start "${url}"`;
```

Comment:
Some people have problems when executing this command (#42). This is why I changed it to

```
start explorer ${url}
```

And by the way I would remove comments like this because it has no use.